sdk/examples/wallet/14_burn_nft.rs

```diff
@@ -13,21 +20,24 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
-    // Create the wallet
-    let wallet = Wallet::builder().finish().await?;
+    // Access the wallet we generated with `--example create_wallet`
+    let wallet = Wallet::builder()
+        .with_storage_path(&var("WALLET_DB_PATH").unwrap())
+        .finish()
+        .await?;
+    let alias = var("ACCOUNT_ALIAS_1").unwrap();
+    let account = wallet.get_account(&alias).await?;
 
-    // Get the account we generated with `01_create_wallet`
-    let account = wallet.get_account("Alice").await?;
     // May want to ensure the account is synced before sending a transaction.
     let balance = account.sync(None).await?;
 
     // Get the first nft
     if let Some(nft_id) = balance.nfts().first() {
-        println!("Balance before burning:\n{balance:?}",);
+        println!("Balance BEFORE burning:\n{balance:?}",);
```

Comment:
lets display only nfts, like the foundry example

Reply:
okay